src/app/visual/responsive/responsive-demo.component.scss

```diff
@@ -0,0 +1,19 @@
+#screen-queries {
+  ::ng-deep .box-1 {
+    background-color: aqua;
+  }
+
+  ::ng-deep .box-2 {
+    background-color: plum;
+  }
+}
+
+#container-queries {
+  ::ng-deep .box-1 {
+    background-color: rgb(255, 102, 0);
+  }
+
+  ::ng-deep .box-2 {
+    background-color: rgb(65, 141, 2);
+  }
+}
```

Comment:
Are these styles really necessary? They don't necessarily need different colors, do they? In the interest of less code, you could eliminate this all-together with some simple headings above each instance of the component like this:
```
Screen queries

[component]

Container queries

[component]
```

Reply:
The colors help show the two boxes easily and makes the visual tests easier to see what is wrong if they fail. That's just my opinion though. The headers are a good idea too though :)

Reply:
Easier to see, sure. I get that. Do the colors really need to be declared from the outer component though? In a normal scenario, wouldn't they be part of `responsive-container.component.scss`?

Also weird question... do visual tests see color?

Reply:
They are in the outer `scss` in order to allow for the inspection of the "container-queries" and "screen-queries" ids that are at this level. I can move them and use "host-context" but it didn't seem to make a ton of difference to me so I just did it here. I think the distinction between the "screen" and "container" sections is a big deal in terms of this test. I don't mind to move the styles if that makes you feel better but I don't want to lose the color difference.

And I double checked this and it did seem to be sensitive to color.

Reply:
Sounds good, thanks for checking.